python/TestHarness/schedulers/QueueManager.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, harness, params):
         self.options = self.harness.getOptions()
         self.__job_storage_file = self.harness.original_storage
         self.__clean_args = None
+        self.__status_system = StatusSystem()
```

Comment:
Just to make sure -- this line (I believe) creates a *new* status system. Is this intentional, or is it supposed to be referencing an existing status system?

Reply:
It's a new one. But if you look at what a `StatusSystem` is - it's really just a data struct that helps you understand statuses